app/server/utils/user.py

```diff
@@ -858,43 +864,43 @@ def attach_transfer_account_to_user(user, organisation=None):
     The method also takes an organization object as an argument to match the user object to a specific organization.
 
     :param user: The initial account the user created when they began the self sign up process.
-    :param organisation: The organization the user belongs to.
     :return: A user with a transfer account,
     """
+
+    if user.primary_blockchain_address:
+        raise Exception('User already has a transfer account attached.')
```

Comment:
swapped to using guard clause because the if statements were getting pretty nested

Reply:
is there exception catching further up?

Reply:
no idea lol! Presumably if these exceptions were written...?